### ISSUE TITLE
[Bug]: incoming notification but no floating notification

### ISSUE THREAD

**Issue by ayyuby26** [OneSignal/OneSignal-Flutter-SDK#609]
### What happened?

in version 3.4.2 the notification came in but only the notification sound, the floating notification didn't come in, but when I checked the notification list, the notification was there, I found this in the official onesignal video tutorial at 4:20 minutes

onesignal official video tutorial: https://youtu.be/5klspCULQe4?t=260

### Steps to reproduce?

```Markdown
1. install v3.4.2 depedency
2. launch the app android
3. send notification
4. incoming notification but no floating notification
```

### What did you expect to happen?

incoming and floating notifications

### OneSignal Flutter SDK version

3.4.2

### Which platform(s) are affected?

- [ ] iOS
- [X] Android

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by chrisvidal** [comment]
having the same question

**Comment by chrisvidal** [comment]
check your code that handles the event `setNotificationWillShowInForegroundHandler`

in the code that I copied/pasted from the website, there was this line that is blocking the badge. just comment it.

```dart
event.complete(null);
```

**Comment by TareqAmenahDev** [comment]
Same issue here

**Comment by Sahad2701** [comment]
Hello @ayyuby26,
                     
To resolve this issue on Android, you can create an Android Notification Channel where you can define the importance of notifications. By defining a high priority, you will receive sound and popup notifications as well. If you are using an API, you need to define that channel ID in Notification API as key ‘**android_channel_id**’ and value as ‘**your_android_channel_id**’, which you can obtain from your OneSignal Dashboard by going to **Settings > Messaging > Android Notification Channels**.
                      
Full documentation link : https://documentation.onesignal.com/docs/android-notification-categories
     
     
     
<img width="694" alt="Screenshot 2023-10-28 at 8 51 31 PM" src="https://github.com/OneSignal/OneSignal-Flutter-SDK/assets/79684384/f8747430-aa8c-4685-b975-5f9d0aea3657">
<img width="507" alt="Screenshot 2023-10-28 at 8 56 39 PM" src="https://github.com/OneSignal/OneSignal-Flutter-SDK/assets/79684384/45f4d2f9-6137-43ff-ba51-2750a1d3d14d">

I hope this will solve your issue.

**Comment by Sahad2701** [comment]
Closes #609
